client/dashboard/sites/scan/index.tsx

```diff
@@ -31,6 +33,22 @@ function SiteScan( { scanTab }: { scanTab: 'active' | 'history' } ) {
 	const router = useRouter();
 
 	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
+	const { data: scan } = useQuery( siteScanQuery( site.ID ) );
+
+	const lastScanTime = scan?.most_recent?.timestamp;
+	const lastScanRelativeTime = useTimeSince( lastScanTime || '' );
+
+	const getPageDescription = () => {
+		if ( lastScanTime && lastScanRelativeTime ) {
+			return sprintf(
+				/* translators: %s: relative time since last scan */
+				__( 'Latest automated scan ran %s' ),
```

Comment:
Nit: p1753935125453589-slack-C08JZTRS6NA 
```suggestion
				__( 'Latest automated scan ran %s.' ),
```